### PR TITLE
[stable/sentry] Add missing values for the db-init and user-create jobs

### LINKED ISSUE #6321: [stable/sentry] Support specifying resources for jobs
**Is this a request for help?**:
Yes
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
FEATURE REQUEST

**Version of Helm and Kubernetes**:
Helm: v2.9.1
Kubernetes: v1.10.1

**Which chart**:
stable/sentry

**What happened**:
When installing stable/sentry on a cluster with the LimitRanger admission controller, the database seed and user creation jobs both failed to launch due to missing `limit.cpu` and `limit.memory` values.

There are no methods to provide said limits through the provided values in the chart.

**What you expected to happen**:
The available values should contain settings to let you specify resource requirements / limits for the db-init and user-create jobs.

**How to reproduce it** (as minimally and precisely as possible):
```
helm install stable/sentry --set <no parameter for job resources>
```

**Anything else we need to know**:
While LimitRange objects can provide some defaults for the resources, they are *not* a solution to the issue, especially since the db-init job seems to require quite a chunk of RAM - it would OOM until I gave it 2GB of memory.

### PR DESCRIPTION
**Pull request by ananace**
**What this PR does / why we need it**:
Adds missing resource configuration - and ability to set selectors, tolerations, affinity, and scheduler name - for the db-init and user-create jobs.
This should make it more possible to deploy sentry on clusters with resource quotas, tainted nodes, and/or other cluster restrictions.

**Which issue this PR fixes**: fixes #6321

### CODE REVIEW

**Comment by insertjokehere** [comment]
@ananace Generally you need to assign PRs to people to get them reviewed, see the comment from k8s-ci-robot above for instructions

File: stable/sentry/values.yaml [hunk 1]
```diff
@@ -63,6 +63,33 @@ worker:
   affinity: {}
   # schedulerName:
 
+# Job resources
+jobs:
+  db_init:
```
Comment (unguiculus) [inline stable/sentry/values.yaml]: Use camel case

File: stable/sentry/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Sentry is a cross-platform crash reporting and aggregation platform.
 name: sentry
-version: 0.4.1
+version: 0.4.2
```
Comment (unguiculus) [inline stable/sentry/Chart.yaml]: Bump minor version

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/sentry/values.yaml (hunk 1 above)
Comment (ananace) [reply stable/sentry/values.yaml]: I wanted to keep the job resource configuration named the same as the job itself, but I can change it to camel case for consistency's sake.

Comment (stephenmoloney) [reply stable/sentry/values.yaml]: camelcase is the kubernetes convention AFAIK.

**Comment by stephenmoloney** [comment]
@ananace 

I was going to make the resources for the job configurable but then I noticed this PR. 
Any help required?

**Comment by ananace** [comment]
Well, if you can help get people to review it so I don't have to keep merging and bumping the version :slightly_smiling_face:

**Comment by stephenmoloney** [comment]
I don't know what the standard practice is for helm but it might be worth squashing these changes into a single commit.

Also, typically I rebase rather than merge master.

**Comment by ananace** [comment]
I typically do new commits during PR reviews, and then squash at the end, unless there's only a tiny requested change with just a few nitpicks - nothing that changes code flow itself.

And I tend to rebase as well, though the GitHub web interface doesn't really allow for that like it does merges.

**Comment by insertjokehere** [comment]
@unguiculus Are you happy with the changes that have been made? This PR has been kicking around for a while, can we get this merged please?

**Comment by ananace** [comment]
Have your requested changes been resolved yet? @unguiculus 

This has been laying about for quite a while now.

**Comment by ananace** [comment]
Still waiting on a response, is the applied changes acceptable @unguiculus?

**Pull request closed without merging**